backend/app/utils/streaming_jsonl_extractor.py

```
@@ -0,0 +1,184 @@
+"""
+流式JSONL内容提取器
+
+在LLM流式输出过程中实时识别和提取JSONL内容，
+忽略markdown代码块标记、多余的文本等干扰信息。
+"""
+
+import re
+import logging
+from typing import Optional, Tuple
```

Ruff (0.11.9)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)